internal/ipfamily/ipfamily.go

```diff
@@ -44,7 +44,7 @@ func ForAddresses(ips []string) (Family, error) {
 		if (ip1.To4() == nil) == (ip2.To4() == nil) {
 			return Unknown, fmt.Errorf("IPFamilyForAddresses: same address family %q", ips)
 		}
-		return DualStack, nil
+		return RequiredDualStack, nil
```

Comment:
this is debatable, or I am inclned to say the ipfamily for a v4 + v6 is dual, not required.